vulnfeeds/cves/versions.go

```diff
@@ -739,6 +743,33 @@ func Commit(u string) (string, error) {
 	return "", fmt.Errorf("Commit(): unsupported URL: %s", u)
 }
 
+// Detect linkrot via HEAD request with exponential backoff.
+func ValidateLink(link string) (err error) {
+	backoff := retry.NewExponential(1 * time.Second)
+	if err := retry.Do(context.Background(), retry.WithMaxRetries(3, backoff), func(ctx context.Context) error {
+		resp, err := http.Head(link)
```

Comment:
Do we need to set any timeout values anywhere?

Reply:
> Do we need to set any timeout values anywhere?

That's line 748 above. I'll add a comment to make that clearer.

Reply:
Sorry, I misunderstood. Looking at https://pkg.go.dev/net/http#Client, it defaults to https://pkg.go.dev/net/http#RoundTripper, which uses a 30 second timeout, which seems reasonable to me.